From: Ryan Lee <ryan.lee@canonical.com>
Date: Tue, 14 Apr 2026 13:15:41 -0700
Subject: [PATCH] profiles: grant unix domain socket access to
 sanitized_helpers

sanitized_helpers is an unfortunate hack, but there are users right now
that are expecting to be able to access unix domain sockets with it.

Fixes: https://bugs.launchpad.net/bugs/2148177

Signed-off-by: Ryan Lee <ryan.lee@canonical.com>
---
 profiles/apparmor.d/abstractions/ubuntu-helpers | 1 +
 1 file changed, 1 insertion(+)

diff --git a/profiles/apparmor.d/abstractions/ubuntu-helpers b/profiles/apparmor.d/abstractions/ubuntu-helpers
index d468878..2d849dc 100644
--- a/profiles/apparmor.d/abstractions/ubuntu-helpers
+++ b/profiles/apparmor.d/abstractions/ubuntu-helpers
@@ -41,6 +41,7 @@ profile sanitized_helper {
   # Allow all networking
   network inet,
   network inet6,
+  network unix,
 
   # Allow all DBus communications
   include <abstractions/dbus-session-strict>
